events/iisc_registry_events.txt

```diff
@@ -57,12 +52,26 @@ country_event = {
 						}
 					}
 				}
-			}
-			ROOT = {
-				change_variable = {
-					which = iisc_registry_length
-					value = 1
+				PREV = {
```

Comment:
I think you already store the HQ just above this?

Reply:
oops